src/main/java/com/web01/animatch/controller/AccountChangeController.java

```diff
@@ -0,0 +1,67 @@
+package com.web01.animatch.controller;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.MultipartConfig;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.web01.animatch.service.AccountChangeService;
+
+/**
+ * AccountChangeControllerクラス
+ * @author Tsuji
+ * @version 1.0
+ */
+@MultipartConfig
+public class AccountChangeController extends HttpServlet {
+
+ //定数
+ /**
+  * シリアライズバージョンID
+  */
+ private static final long serialVersionUID = 1L;
+
+ /**
+  * デフォルトコンストラクタ
+  */
+ public AccountChangeController() {
+  super();
+ }
+
+ /**
+  * get送信
+  * @param request リクエストオブジェクト
+  * @param response レスポンスオブジェクト
+  */
+ protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+  AccountChangeService AccountChangeService = new AccountChangeService();
+  AccountChangeService.setFirstDisplay(request);
+
+  String path = "/WEB-INF/jsp/account/change/change.jsp";
+  RequestDispatcher dispatcher = request.getRequestDispatcher(path);
+  dispatcher.forward(request, response);
+ }
+
+ /**
+  * post送信
+  * @param request リクエストオブジェクト
+  * @param response レスポンスオブジェクト
+  */
+ protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+  AccountChangeService AccountChangeService = new AccountChangeService(request.getParameter("formRegistType"));
+  if (AccountChangeService.change(request)) {
+   String path = "/WEB-INF/jsp/account/change/change_complete.jsp";
+   RequestDispatcher dispatcher = request.getRequestDispatcher(path);
+   dispatcher.forward(request, response);
+  } else {
+   //登録に失敗した場合
+   String path = "/WEB-INF/jsp/account/change/change.jsp";
+   RequestDispatcher dispatcher = request.getRequestDispatcher(path);
+   dispatcher.forward(request, response);
+  }

```

Comment:
pathの設定がif~elseの役割のようなので、重複部分は外に出してしまってよいと思いました。

```suggestion
  String path = "/WEB-INF/jsp/account/change/change.jsp";
  if (AccountChangeService.change(request)) {
   path = "/WEB-INF/jsp/account/change/change_complete.jsp";
  }
  RequestDispatcher dispatcher = request.getRequestDispatcher(path);
  dispatcher.forward(request, response);
```

Reply:
上記、承知いたしました。修正いたします。

Reply:
修正を確認致しました。